src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -455,7 +457,7 @@ public void setLabels(Map<String, String> labels) {
      */
     @NonNull
     public List<PodLabel> getPodLabels() {
-        return podLabels == null || podLabels.isEmpty() ? PodLabel.fromMap(DEFAULT_POD_LABELS) : podLabels;
+        return podLabels == null ? List.of() : podLabels;
```

Comment:
The previous idiom made more sense to me actually. It depends on whether `DEFAULT_POD_LABELS` might ever change in the future, and if so, what we expect the impact to be on existing templates. https://www.jenkins.io/doc/developer/plugin-development/pipeline-integration/#handling-default-values has some discussion. At any rate, what about the previous handling of `podLabels` did not round-trip correctly?